src/styles/annotator/annotator.scss

```diff
@@ -16,12 +19,14 @@
 @use './components/NotebookModal';
 @use './components/Toolbar';
 @use './components/WarningBanner';
-@use './highlights';
 
 // Sidebar
 .annotator-frame {
   // CSS reset which attempts to isolate this element and its children from
   // host page styles.
+  //
+  // TODO - We should be able to remove these now because this element is always
+  // created inside a shadow root.
   @include meta.load-css('../reset');
   @include reset.nested-reset;
   @include reset.reset-box-model;
```

Comment:
It should be possible to replace this with our standard CSS reset import in future.